addons/udes_stock/wizard/change_quant_location_view.xml

```diff
@@ -26,5 +26,24 @@
             </field>
         </record>
 
+        <record id="view_move_reserved_package_check" model="ir.ui.view">
+            <field name="name">Create Picking for Reserved Package(s)?</field>
+            <field name="model">change_quant_location</field>
```

Comment:
Both `view_move_reserved_package_check` and `create_picking_for_packages_form` are form views for `change_quant_location`, but neither one is set explicitly as a default.

Could you set a priority on the least important one? As long as the priority is above the default 16 it should be fine.

e.g. `<field name="priority">20</field>`